src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -726,6 +741,11 @@
 /* Local Notifications */
 "local_notifications_viewResults" = "Display Results";
 
+// App Information - Legal
```

Comment:
Aren't we supposed to not change the english translations files anymore? Just making sure to be on the safe side.

And I think we should also use /* */ in the english file as well.

Reply:
I'll revert these in the next commit